pynetbox/models/dcim.py

```diff
@@ -154,7 +222,21 @@ def __str__(self):
 
 class Cables(Record):
     def __str__(self):
-        return "{} <> {}".format(self.termination_a, self.termination_b)
+        # populate the terminations to get the full names if they are not already
```

Comment:
this is more of a bugfix/enhancement, it's related but I can split it out to a dedicated PR if you want

Reply:
Sure, with this section of code reverted this is what I'm getting now:

```
In [1]: iface.trace()                                                                                                             
Out[1]: 
[[em1,
  <class 'pynetbox.models.dcim.Termination'> <> <class 'pynetbox.models.dcim.Termination'>,
  pair-11 (ports 21-22)],
 [port-2,
  <class 'pynetbox.models.dcim.Termination'> <> <class 'pynetbox.models.dcim.Termination'>,
  port-2],
 [pair-11 (ports 21-22),
  <class 'pynetbox.models.dcim.Termination'> <> <class 'pynetbox.models.dcim.Termination'>,
  Ethernet11]]

```

So it looks like maybe I was mistaken or confused about only the middle (not the first or last) having the issue, it may be all of the items returned from the trace until they are accessed in some way. You can see here how the info is filled out once the item is accessed directly: 
```
In [2]: trace = iface.trace()                                                                                                     

In [3]: trace                                                                                                                     
Out[3]: 
[[em1,
  <class 'pynetbox.models.dcim.Termination'> <> <class 'pynetbox.models.dcim.Termination'>,
  pair-11 (ports 21-22)],
 [port-2,
  <class 'pynetbox.models.dcim.Termination'> <> <class 'pynetbox.models.dcim.Termination'>,
  port-2],
 [pair-11 (ports 21-22),
  <class 'pynetbox.models.dcim.Termination'> <> <class 'pynetbox.models.dcim.Termination'>,
  Ethernet11]]

In [4]: trace[0][1]                                                                                                               
Out[4]: em1 <> pair-11 (ports 21-22)

In [5]: trace                                                                                                                     
Out[5]: 
[[em1, em1 <> pair-11 (ports 21-22), pair-11 (ports 21-22)],
 [port-2,
  <class 'pynetbox.models.dcim.Termination'> <> <class 'pynetbox.models.dcim.Termination'>,
  port-2],
 [pair-11 (ports 21-22),
  <class 'pynetbox.models.dcim.Termination'> <> <class 'pynetbox.models.dcim.Termination'>,
  Ethernet11]]

```